01_prepare_host.sh

```diff
@@ -23,6 +23,7 @@ if [[ $OS == ubuntu ]]; then
   fi
 
 else
+  sudo dnf install -y epel-release && sudo dnf update -y
```

Comment:
I have  modifications for this, that are part of the Ansible scripts, I will push it so this would be a bit redundant.

Reply:
I have actually added this to ansible centos package install section. I haven't pushed it yet but do we want it in shell or in ansible?

Reply:
Okay it was discussed offline.

Reply:
I will just reply here too for the git history. I just added it here so that we make sure to install it before actually going to install packages listed in the Ansible. But as said I don't have a strong opinion, and it can be added in Ansible too.